src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -210,7 +210,7 @@ public override INamedTypeSymbol CreateErrorTypeSymbol(INamespaceOrTypeSymbol co
         /// </summary>
         public static CSharpCompilation CreateSubmission(
             string assemblyName,
-            SyntaxTree syntaxTree = null,
+            IEnumerable<SyntaxTree> syntaxTrees = null,
```

Comment:
Do we want to provide a convenience overload that accepts a single tree?  @AnthonyDGreen